samples/client/echo_api/java/okhttp-gson/src/test/java/org/openapitools/client/model/BirdAndCategoryTest.java

```diff
@@ -34,15 +34,19 @@ public class BirdAndCategoryTest {
      */
     @Test
     public void testBirdAndCategory() {
-        // TODO: test BirdAndCategory
+        birdAndCategory.setSize("Small");
+        assertEquals("Small", birdAndCategory.getSize());
+
+        birdAndCategory.setSize("Large");
```

Comment:
should be `model` instead of `birdAndCateogry` ? 

same for line 37

Reply:
Yeah let me change and push it.